connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -185,10 +192,10 @@ public void close() {
         deleteAllTopics(backup.kafka());
         primary.stop();
         backup.stop();
+        assertFalse(exited.get());
+        Exit.resetExitProcedure();
```

Comment:
I am guessing we ran into an issue because of exit, so perhaps this reset ought to be in a finally block?

Reply:
@rajinisivaram good point. I'll fix that.

Reply:
I've pushed a fix. Thanks.